amcl/CMakeLists.txt

```diff
@@ -108,24 +108,24 @@ if(CATKIN_ENABLE_TESTING)
 #    http://download.ros.org/data/amcl/texas_greenroom_loop_indexed.bag
 #    DESTINATION ${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_SHARE_DESTINATION}/test
 #    MD5 6e3432115cccdca1247f6c807038e13d)
-  catkin_download_test_data(${PROJECT_NAME}_texas_willow_hallway_loop_indexed.bag
-    http://download.ros.org/data/amcl/texas_willow_hallway_loop_indexed.bag
-    DESTINATION ${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_SHARE_DESTINATION}/test
-    MD5 27deb742fdcd3af44cf446f39f2688a8)
-  catkin_download_test_data(${PROJECT_NAME}_rosie_localization_stage.bag
-    http://download.ros.org/data/amcl/rosie_localization_stage.bag
-    DESTINATION ${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_SHARE_DESTINATION}/test
-    MD5 3347bf3835724cfa45e958c5c1846066)
+#  catkin_download_test_data(${PROJECT_NAME}_texas_willow_hallway_loop_indexed.bag
+#    http://download.ros.org/data/amcl/texas_willow_hallway_loop_indexed.bag
+#    DESTINATION ${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_SHARE_DESTINATION}/test
+#    MD5 27deb742fdcd3af44cf446f39f2688a8)
+#  catkin_download_test_data(${PROJECT_NAME}_rosie_localization_stage.bag
+#    http://download.ros.org/data/amcl/rosie_localization_stage.bag
+#    DESTINATION ${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_SHARE_DESTINATION}/test
+#    MD5 3347bf3835724cfa45e958c5c1846066)
 
   # Maps
   catkin_download_test_data(${PROJECT_NAME}_willow-full.pgm
     http://download.ros.org/data/amcl/willow-full.pgm
     DESTINATION ${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_SHARE_DESTINATION}/test
     MD5 b84465cdbbfe3e2fb9eb4579e0bcaf0e)
-  catkin_download_test_data(${PROJECT_NAME}_willow-full-0.05.pgm
-    http://download.ros.org/data/amcl/willow-full-0.05.pgm
-    DESTINATION ${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_SHARE_DESTINATION}/test
-    MD5 b61694296e08965096c5e78611fd9765)
+#  catkin_download_test_data(${PROJECT_NAME}_willow-full-0.05.pgm
```

Comment:
why are we commenting out the tests

Reply:
we aren't. This was just downloading data used for tests that we don't run anymore